### DISCUSSION TITLE
Is this preamp feasible to simulate in Livespice?

### DISCUSSION THREAD

**Discussion by joshdevvv** [dsharlet/LiveSPICE#240] [Q&A] [answered]
Managed to find this schematic of a preamp, I know basically nothing about amp building, very little about electronics, and haven't worked much in Livespice yet. Figured I would ask before I try to pour too much time into trying to recreate it. If it does approximately work, even if the latency is huge it could be productive to just train a NAM model on it

![rt-922-1](https://github.com/user-attachments/assets/28217f38-9d8a-49db-ab95-f623df6d07c6)
![rt-922-2](https://github.com/user-attachments/assets/c1625426-a7f2-43db-8863-aa58296ed08a)

**Comment by dsharlet** [comment, marked as answer]
I think parts of this can definitely be simulated, but trying to simulate everything here at once is going to be tough, very unlikely to be real time. There are a *lot* of diodes in that second schematic!

The first schematic looks quite doable. We have other examples that use 3 or 4 tubes and that simulate well. A lot depends on how you choose to model the op-amps. If you use an ideal opamp model (no power supply terminals), the upper schematic looks doable.

A lot of our schematics omit parts of the circuit that I or someone else determined aren't significant to the result in the ideal world of a simulation. You might want to do that here too. For example, I think many of the diodes in the second schematic are LEDs that might not affect the sound simulation significantly.
